core/unix/coredump.c

```diff
@@ -45,6 +49,17 @@
 #define SECTION_HEADER_TABLE ".shstrtab"
 #define VVAR_SECTION "[vvar]"
 #define VSYSCALL_SECTION "[vsyscall]"
+// The name of the note owner including the terminating null character needs to
+// be 4 byte aligned.
+#define NOTE_OWNER "CORE\0\0\0\0"
```

Comment:
Why "CORE"?  Does this mean something: is it a convention for core dump files?  Add a comment.

Reply:
Yes, I followed the owner name in core dump files.